aries_cloudagent/tails/indy_tails_server.py

```diff
@@ -10,13 +10,23 @@ class IndyTailsServer(BaseTailsServer):
     """Indy tails server interface."""
 
     async def upload_tails_file(
-        self, context, rev_reg_id: str, tails_file_path: str
+        self,
+        context,
+        rev_reg_id: str,
+        tails_file_path: str,
+        interval: float = 1.0,
+        backoff: float = 0.25,
```

Comment:
It may just be me not being familiar with this, but these interval and backoff do not appear to be used: this would cause the attempts to be sequentially kicked-off based on the time it took to complete (fail) the previous request.

Reply:
Based on the comment to `aries_cloudagent/utils/http.py ` it then looks like these are in fact not being used, and the default values defined for the utility method are used instead.